src/openrct2-ui/windows/EditorInventionsList.cpp

```diff
@@ -618,7 +618,7 @@ class InventionDragWindow final : public Window
         {
             res = inventionListWindow->GetResearchItemAt(newScreenCoords);
             newScreenCoords.y += LIST_ROW_HEIGHT;
-        } while (res.has_value() && res->research->IsAlwaysResearched());
+        } while (res->research != nullptr && res->research->IsAlwaysResearched());
```

Comment:
You should still check has value

Reply:
Readd has_check() to the while condition